### ISSUE TITLE
feat: add versioning

### ISSUE THREAD

**Issue by toddeTV** [toddeTV/zlig#74]
versioning approaches:

- ~~date based, e.g. `20240930`~~
- ~~commit hash, e.g. `a1b2c3d`~~
- semantic versioning, e.g. `v2.1.7`
- timestamp based, e.g. `v2025.118.151840`

~~Or combinations of the above.~~

& CI

**Comment by fehnomenal** [comment]
Yeah, I think date/timestamp-based versioning is a good fit for a website/app that is centrally available. So go for it if you have capacity :+1:
